email/parser.go

```diff
@@ -93,6 +104,23 @@ func parseMessageWithHeader(headers Header, bodyReader io.Reader) (*Message, err
 	}, nil
 }
 
+// removeDuplicate removes duplicate from bogus content type preventing parsing of email
+func removeDuplicate(s string) string {
+	m := make(map[string]bool)
+	parts := strings.Split(s, ";")
+	for _, p := range parts {
+		if _, ok := m[p]; !ok {
+			m[p] = true
+		}
+	}
+
+	var cleaned string
+	for k := range m {
+		cleaned += k + ";"
```

Comment:
Do we need a space after the `;`?
Also, you can probably just return `strings.Join(m, "; ")` as that is both cleaner and faster